cost/aws/rightsize_rds_instances/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v5.8.1
+
+- Improved accuracy of savings calculation for underutilized RDS instances recommending to be resized.  We now consider only the "InstanceUsage" usage type costs when estimating potential savings from resize.
```

Comment:
Nitpicky, I know, but should only be one space between "resized." and "We"

Reply:
```suggestion
- Improved accuracy of savings calculation for underutilized RDS instances recommending to be resized. We now consider only the "InstanceUsage" usage type costs when estimating potential savings from resize.
```